auto_submit/lib/validations/unknown_mergeable.dart

```diff
@@ -16,7 +17,10 @@ class UnknownMergeable extends Validation {
   @override
   Future<ValidationResult> validate(QueryResult result, github.PullRequest messagePullRequest) async {
     // This is used to skip landing until we are sure the PR is mergeable.
-    final bool unknownMergeableState = messagePullRequest.mergeable == null;
+    final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
+    final GithubService githubService = await config.createGithubService(slug);
+    final github.PullRequest updatedPr = await githubService.getPullRequest(slug, messagePullRequest.number!);
+    final bool unknownMergeableState = updatedPr.mergeable == null;
```

Comment:
Shall we do this only when the `messagePullRequest.mergeable == null`? Otherwise seems no need to double query the pull request.

Reply:
Hang on. I might have a better way to do this which not result in another query.